app/models/color.rb

```diff
@@ -13,17 +13,14 @@ def self.black
     where(name: "Black", priority: 1, display: "#000").first_or_create
   end
 
-  def self.name_translation(name, locale: nil)
-    I18n.t(
-      name.to_s.downcase.gsub(/[^[:alnum:]]+/, "_"),
-      scope: [:activerecord, :select_options, self.name.underscore],
-      locale: locale,
-    )
-  end
+  def self.select_options
+    normalize = ->(value) { value.to_s.downcase.gsub(/[^[:alnum:]]+/, "_") }
```

Comment:
What's the impact on load times for this? Do we need to do caching?

Reply:
Negligible, and caching introduces some complexity with testing since this is a class method.